omniscidb/ResultSet/ResultSet.cpp

```diff
@@ -1055,7 +1055,7 @@ size_t ResultSet::computeVarLenOffsets(size_t col_idx, int32_t* offsets) const {
   }
 
   // Translate varlen value to its length. Return -1 for NULLs.
-  auto slot_val_to_length = [this, lazy_fetch, col_idx, type, target_info](
+  auto slot_val_to_length = [this, lazy_fetch, col_idx, type, target_info, arr_elem_size](
```

Comment:
I'd change the name since you've modified it's meaning.

Reply:
I think the name still correctly describes the method's meaning. It gets a slot value and returns the corresponding array length in bytes.